README.md

```diff
@@ -378,6 +388,40 @@ Note, that the parameter must follow the format "uid:gid" where `uid` and `gid`
 
 Allows specifying the Python 3 interpreter to be used when calling the Pieman-specific utilities. 
 
+#### Extra
+
+##### CREATE_ONLY_ARTIFACT=false
```

Comment:
How about `MENDER_CREATE_ONLY_ARTIFACT`?

Reply:
I wanted it to look like another similar param -- `CREATE_ONLY_CHROOT`.

Reply:
Renamed it into `CREATE_ONLY_MENDER_ARTIFACT`.